node/src/Commands.ts

```diff
@@ -70,39 +78,46 @@ export function createGet(key: string): redis_request.Command {
 
 export type SetOptions = {
     /**
-     *  `onlyIfDoesNotExist` - Only set the key if it does not already exist. Equivalent to `NX` in the Redis API.
-     * `onlyIfExists` - Only set the key if it already exist. Equivalent to `EX` in the Redis API.
-     * if `conditional` is not set the value will be set regardless of prior value existence.
-     * If value isn't set because of the condition, return null.
+     *  `onlyIfDoesNotExist` - Only set the key if it does not already exist.
```

Comment:
Please revert all unrelated stylying changes. Maybe re-running linters would help.

Reply:
This change was done by running the linter... I did not change this manually
If a linter should not be run - remove it from the docs. Otherwise, we will keep hitting this style change. I suggest to keep this change, as we are bound to get this change in future PRs

Reply:
All of the files were lint into a unified standard - the default standard of prettier. If your linter changed the file it means that your linter is set with different settings from the default.
Please try to reinstall prettier and make sure all of your settings are set to default settings.

Reply:
@eifrah-aws See Avi's comment